source/extensions/transport_sockets/alts/config.cc

```diff
@@ -26,6 +26,15 @@ using GrpcAltsCredentialsOptionsPtr =
 
 namespace {
 
+// There is an equivalent function grpc_alts_set_rpc_protocol_versions in newer
+// release of gRPC that should be called directly when available.
```

Comment:
Thanks! Could you add which gRPC version with a TODO?

Reply:
Added explicit TODO with a version.